src/main/java/org/jvnet/hudson/test/InboundAgentRule.java

```diff
@@ -80,7 +80,7 @@
  * </pre>
  *
  * @see JenkinsRule#createComputerLauncher
- * @see JenkinsRule#createSlave()
+ * @see JenkinsRule#createAgent()
```

Comment:
There is no [createAgent](https://javadoc.jenkins.io/component/jenkins-test-harness/org/jvnet/hudson/test/JenkinsRule.html) method, so this link will fail to resolve
```suggestion
 * @see JenkinsRule#createSlave()
```